pkg/controllers/namespace.go

```diff
@@ -83,7 +83,6 @@ func (c *NamespaceConfig) Run(stopCh <-chan struct{}) {
 	}
 
 	for i := range c.eventHandlers {
-		klog.V(4).Infof("Calling handler.OnNamespaceSynced()")
```

Comment:
I'd like to know why you remove this. This is sometimes important to check callback function (especially complicated troubleshooting) but I understand that you feel that is redundant on the other side. 

How about to move to to 'V(10)' or some?

Reply:
Ok, for `V(10)` is a good tradeoff. Otherwise raising a little bit the log level (to 4actually), turns in a lot of duplicated logs like:

```
I0524 15:12:25.914324       1 namespace.go:86] Calling handler.OnNamespaceSynced()
I0524 15:12:25.914333       1 server.go:421] OnNamespaceSynced
```

As `Server` is the only one that contains production implementation of `NamespaceHandler`. The same logic applies for all other kind of handlers.